bats/tests/k8s/up-downgrade-k8s.bats

```diff
@@ -0,0 +1,96 @@
+# Test case 8 & 13
+
+# shellcheck disable=SC2030,SC2031
+# SC2030 (info): Modification of output is local (to subshell caused by @bats test).
+# SC2031 (info): output was modified in a subshell. That change might be lost.
+
+setup() {
+    load '../helpers/load'
+}
+
+@test 'factory reset' {
+    factory_reset
+}
+
+@test 'start rancher desktop' {
+    start_kubernetes
+    wait_for_apiserver
+    # the docker context "rancher-desktop" may not have been written
+    # even though the apiserver is already running
+    wait_for_container_engine
+}
+
+@test 'deploy nginx - always restart' {
+    ctrctl pull nginx
+    run ctrctl run -d -p 8585:80 --restart=always --name nginx-restart nginx
+    assert_success
+}
+
+#@test 'deploy nginx - no restart' {
+ #   ctrctl pull nginx
+  #  ctrctl run -d -p 8686:80 --restart=no --name nginx-no-restart nginx
+#}
+
```

Comment:
Please remove commented out code, or uncomment it, and add tests for the `--restart=no` case after upgrading Kubernetes.